beacon-chain/db/db.go

```diff
@@ -6,9 +6,22 @@ import (
 	"context"
 
 	"github.com/prysmaticlabs/prysm/beacon-chain/db/kv"
+	"github.com/prysmaticlabs/prysm/beacon-chain/db/slasherkv"
 )
 
 // NewDB initializes a new DB.
 func NewDB(ctx context.Context, dirPath string, config *kv.Config) (Database, error) {
 	return kv.NewKVStore(ctx, dirPath, config)
 }
+
+// NewDBFilename uses the KVStoreDatafilePath so that if this layer of
+// indirection between db.NewDB->kv.NewKVStore ever changes, it will be easy to remember
+// to also change this filename indirection at the same time.
+func NewDBFilename(dirPath string) string {
+	return kv.KVStoreDatafilePath(dirPath)
+}
+
+// NewSlasherDB initializes a new DB for slasher.
```

Comment:
@kasey slasher isn't fully merged yet so you might run into problems here. Let's remove until #8340 is done

Reply:
actually this should be ok, but would need bazel run //:gazelle -- fix

Reply:
rebase and mod cleanup is done, PTAL